packaging/ubuntu/copyright

```diff
@@ -1,8 +1,8 @@
-Files: tracee/tracee-ebpf/tracee.bpf.c
+Files: tracee/pkg/ebpf/c/tracee.bpf.c
 Copyright: 2022, Aqua Security
 License: GPL-2+
 
-Files: tracee/tracee-ebpf/co-re/*
+Files: tracee/tracee-ebpf/c/*
```

Comment:
Should probably be `tracee/pkg/ebpf/c/*` and then the above `tracee/pkg/ebpf/c/tracee.bpf.c` copyright can be removed